save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/entities/ContestResult.kt

```diff
@@ -8,11 +11,20 @@ import kotlinx.serialization.Serializable
  * @property organizationName name of an organization in which given project is
  * @property score that project got in contest
  * @property contestName
+ * @property submissionTime
+ * @property submissionStatus
+ * @property sdk
+ * @property hasFailedTest
  */
 @Serializable
 data class ContestResult(
     val projectName: String,
     val organizationName: String,
     val contestName: String,
-    val score: Double?,
+    val score: Double? = null,
+    @Contextual
+    val submissionTime: LocalDateTime? = null,
```

Comment:
Can we use `kotlinx.datetime.LocalDateTime` directly; without `@Contextaul` and our typealias?

Reply:
Without any additional measures Jackson and kotlinx.serialization work bad together, won't fix

Reply:
I'll mention this in #908 so that we don't forget it once again :)